accumulo/src/main/scala/geotrellis/spark/io/accumulo/AccumuloAttributeStore.scala

```diff
@@ -111,18 +114,22 @@ class AccumuloAttributeStore(val connector: Connector, val attributeTable: Strin
 
   def layerIds: Seq[LayerId] = {
     val scanner = connector.createScanner(attributeTable, new Authorizations())
-    scanner.iterator
+    val retval = scanner.iterator
       .map { kv =>
         val List(name, zoomStr) = kv.getKey.getRow.toString.split(SEP).toList
         LayerId(name, zoomStr.toInt)
       }
       .toList
       .distinct
+
+    scanner.close ; retval
```

Comment:
A more safe way to write this would be 
```scala
  val scanner = connector.createScanner(attributeTable, new Authorizations())
  try {
   scanner.iterator	
        .map { kv =>
          val List(name, zoomStr) = kv.getKey.getRow.toString.split(SEP).toList
          LayerId(name, zoomStr.toInt)
        }
        .toList
        .distinct
  } finally {
    scanner.close()
  }
```

The rule generally followed with parans or no parans on a method that takes no arguments is that, if the method has side effects, include the parans so that it would never be mistaken for a `val`